vscode/src/main.ts

```diff
@@ -224,123 +233,57 @@ const register = async (
         )
     }
 
+    // Initialize enterprise context
     const enterpriseContextFactory = new EnterpriseContextFactory(completionsClient)
-    disposables.push(enterpriseContextFactory)
+    disposables.push(
+        enterpriseContextFactory,
+        configWatcher.onChange(async _config => {
```

Comment:
If we don't need it, we can skip the argument declaration.

```suggestion
        configWatcher.onChange(async () => {
```